libvast/include/vast/min_max_synopsis.hpp

```diff
@@ -80,12 +80,11 @@ class min_max_synopsis : public synopsis {
     return sizeof(min_max_synopsis);
   }
 
-  caf::error serialize(caf::serializer& sink) const override {
-    return sink(min_, max_);
-  }
-
-  caf::error deserialize(caf::deserializer& source) override {
-    return source(min_, max_);
+  caf::error inspect(supported_inspectors& inspector) override {
+    return std::visit(detail::overload{[this](auto inspector) {
+                        return inspector(min_, max_);
+                      }},
+                      inspector);
```

Comment:
```c++
const auto f = [this](auto& inspector) {
  return inspector(min_, max_);
};
return std::visit(f, inspector);
```

Three things in this:
- Take `inspector` by reference rather than making a copy.
- Remove the overload—it's not necessary if there's just a single lambda.
- Put the visitor into a separate variable to avoid the additional indentation for readability.